CONTRIBUTING.md

```diff
@@ -23,6 +23,18 @@ Use your best judgement, and feel free to propose changes to this document in a
 [raylib Wiki](https://github.com/raysan5/raylib/wiki) contains some information about the library and is open to anyone for edit. 
 Feel free to review it if required, just take care not to break anything.
 
+### raylib C coding conventions
+
+Despite being written in C, raylib does not follow the standard Hungarian notation for C, 
```

Comment:
If there's a standard notation for C, it would be what the standard library and by extension Unix uses, which is `snake_case`. Hungarian notation is more of a DOS/Windows thing, I believe.

Reply:
Oh, never mind me. You just shifted that text around :D

Reply:
raysan is pretty much a Windows guy I think, so it makes sense that he would say it that way. Good info though, thanks!